ruff.hcl

```diff
@@ -2,12 +2,30 @@ description = "Ruff - An extremely fast Python linter, written in Rust."
 test = "ruff --version"
 binaries = ["ruff"]
 
-platform "darwin" {
-  source = "https://github.com/charliermarsh/ruff/releases/download/v${version}/ruff-${xarch}-apple-darwin.tar.gz"
-}
+version "0.5.0" "0.5.1" "0.5.2" "0.5.3" "0.5.4" {
+  platform "darwin" {
+    source = "https://github.com/astral-sh/ruff/releases/download/${version}/ruff-${xarch}-apple-darwin.tar.gz"
```

Comment:
so change in url patter for future versions. hmmm. I would suggest move the source up top so that future version gets picked up will be following this source pattern.

Reply:
an example in bash.

https://github.com/cashapp/hermit-packages/blob/master/bash.hcl#L3

Reply:
This is kinda related to the below symlink in the unpack, so you mean the future version won't get picked up with this way even there is `auto-version` block inside it?

Reply:
putting this source line up to the top level will make the source pattern the current pattern for `ruff` package. 

It's easier for other people to maintain this manifest file in the future too.